portable/ThirdParty/GCC/Posix/port.c

```diff
@@ -340,7 +343,7 @@ static uint64_t prvGetTimeNs( void )
 
     clock_gettime( CLOCK_MONOTONIC, &t );
 
-    return t.tv_sec * 1000000000ULL + t.tv_nsec;
+    return (uint64_t)t.tv_sec * 1000000000ULL + (uint64_t)t.tv_nsec;
```

Comment:
```suggestion
    return ( uint64_t ) t.tv_sec * 1000000000ULL + ( uint64_t ) t.tv_nsec;
```

Reply:
See Comment: https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/620#discussion_r1104071411

Reply:
This also I do not see.